walletdatabase/migrations/sql/1721136888_recreate_indices_balance_history_remove_dups.up.sql

```diff
@@ -1,7 +1,8 @@
 -- Step 1: Create a temporary table to store unique records
 CREATE TABLE IF NOT EXISTS balance_history_temp AS
-SELECT DISTINCT chain_id, address, currency, block, timestamp, balance
-FROM balance_history;
+SELECT DISTINCT chain_id, address, currency, block, MAX(timestamp) as timestamp, balance
```

Comment:
I see the `balance` is not selected which might be a undefined behavior and not have the balance from the `MAX(timestamp)`, or do we have a guarantee that the balance is unique?